etc/kayobe/inventory/group_vars/controllers/network-interfaces

```diff
@@ -29,6 +29,8 @@ alaska_bdn_interface: brhte
 alaska_bdn_bridge_ports:
   - enp3s0f1
 
+dns_interface: "{{ ilab_interface }}"
```

Comment:
I don't think this is necessary. Kayobe uses the `public` network for DNS, and passes the interface on that network through to kolla-ansible as `dns_interface`.

Reply:
The docs probably need a sentence [here](https://kayobe.readthedocs.io/en/latest/configuration/network.html#network-role-configuration) to say that the public network is used for DNS.

Reply:
Good point, thanks.